File: Tanmay Newatia/Day 8/151 - Reverse Words in a String.py
class Solution:
    def reverseWords(self, s: str) -> str:
        s=s.split()
        s.reverse()
        d=""
        for i in range(0,len(s)):
            d+=s[i]
            if i!=len(s)-1:
                d+=" "
        return d 

File: Tanmay Newatia/Day 8/151 - Reverse Words in a String.cpp
class Solution {
public:
    string reverseWords(string s) {
        vector<string> str;
        string word="";
        for(auto i:s){
            if(isblank(i)){
                if(word!=""){
                    str.push_back(word);
                    word="";
                }
                continue;
            }
            else{
                word+=i;
            }
        }
        if (word!=""){
            str.push_back(word);
        }
        string news="";
        for(int i=str.size()-1;i>=0;i--){
            news+=str[i];
            if(i!=0){
                news+=" ";
            }
        }
        return news;
    }
};